explorer/src/api/constants.ts

```diff
@@ -2,6 +2,7 @@
 // master APIs
 export const MASTER_URL = 'https://testnet-milhon-explorer.nymtech.net';
 export const MASTER_VALIDATOR_URL = 'https://testnet-milhon-validator1.nymtech.net';
+export const MAP_TOPOJSON = 'https://raw.githubusercontent.com/zcreativelabs/react-simple-maps/master/topojson-maps/world-110m.json';
```

Comment:
Is this the outline vector for the world map?

If so, I feel it would be better to bundle (or make it a static asset) it because if that content moves / is deleted on GitHub the site could break. The values in there are constant and don't need to be dynamic.